util/processadorProtocolo.py

```diff
@@ -48,9 +48,15 @@ def processaConexao(sConexao):
             # NOTE - a conexão *não deve ser encerrada nesse caso*.
             # NOTE - pular para o proximo email. (ordemComando = 0)
             # !SECTION
-            ordemComando += 1
             emailDestinatario = mensagemTokens[2]
-            sConexao.send('250 {} Recipient ok'.format(emailDestinatario).encode('UTF8'))
+            try:
+                arq = open(emailDestinatario + '.txt', 'r')
```

Comment:
vamos considerar que o nome do arquivo contem apenas os caracteres antes do '@'.
```suggestion
                arq = open(emailDestinatario.split("@")[0] + '.txt', 'r')
```